src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/ProjectHotReloadSession.cs

```diff
@@ -39,6 +39,16 @@ internal class ProjectHotReloadSession : IManagedHotReloadAgent, IProjectHotRelo
 
         public string Name { get; }
 
+        /// <inheritdoc />
+        public IDeltaApplier? DeltApplier
+        {
+            get
+            {
+                EnsureDeltaApplierforSession();
```

Comment:
>EnsureDeltaApplierforSession [](http://example.com/codeflow?start=16&length=28)

.Netfx design guidelines expect property getters to be side-effect free. Is this truly necessary here? What is the impact of EnsureDeltaApplierforSession? Would a method to do this be a better approach?

Reply:
I'll remove this since the places where a session is started already call this function.